modules/csv/docs/api-reference/csv-loader.md

```diff
@@ -26,7 +26,7 @@ const data = await load(url, CSVLoader, {csv: options});
 | Option                  | Type              | Default                 | Description                                                                                                                                                                                                                                                                                     |
 | ----------------------- | ----------------- | ----------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | `csv.header`            | Boolean \| String | `auto`                  | If `true`, the first row of parsed data will be interpreted as field names. If `false`, the first row is interpreted as data.                                                                                                                                                                   |
-| `csv.rowFormat`         | String            | `auto`                  | Can be set to `'object'` go force rows to be objects, or `'auto'`.                                                                                                                                                                                                                              |
+| `csv.rowFormat`         | String            | `auto`                  | Can be set to `'object'` to force rows to be objects, or `'auto'`.                                                                                                                                                                                                                              |
```

Comment:
fixed typo

Reply:
This option specifies if rows should be returned as arrays or objects. This seems unrelated to whether the file has a header or not (we generate `column-#` column names if no header).

I do agree that the `auto` option probably does not make much sense. The app wants to know (i.e. be able to specify) what format it gets from the loader and having to switch based on what was returned is not ideal.

It is reasonable to diverge from Papaparse here and support more firm semantics.

We are getting ready to launch loaders.gl 3.0 so this is a great time to make breaking changes. Just make sure to add a short bullet to the upgrade guide, and perhaps some notes to the whats-new page.

Also see https://github.com/visgl/loaders.gl/issues/1358 that was opened.

Reply:
Thanks for the context! Please note that the additional confusion here is that `rowFormat` isn't used if calling `parse`, but used in `parseInBatches`. We probably should treat this option consistently for both of those methods, no?

We probably should address this in a different PR and keep this one about duplicate columns.